src/App.js

```diff
@@ -34,6 +65,7 @@ function App() {
         <div className="Sudoku__row">{renderCellRow(7)}</div>
         <div className="Sudoku__row">{renderCellRow(8)}</div>
```

Comment:
Couldn't you do that simpler maybe with a table, because to me this looks kind of repetitive.
If not you should substitute the divs with sections to be more semantic.

Reply:
Yeah I planned to change that. But great reminder to use a section:)